fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -68508,6 +68508,167 @@ synonym: "mNSC" NARROW [FlyBase:FBrf0184236]
 intersection_of: FBbt:00005130 ! neurosecretory neuron
 intersection_of: has_soma_location FBbt:00005802 ! pars intercerebralis
 
+[Term]
+id: FBbt:00047190
+name: larval feedforward local interneuron a
+def: "Larval local interneuron that is presynaptic to Basin projection neurons and postsynaptic to mechanosensory chordotonal neurons. The 'a' subgroup preferentially synapse with Basin-2 rather than Basin-1." [FlyBase:FBrf0233796]
+synonym: "iLNa" EXACT [FlyBase:FBrf0233796]
+is_a: FBbt:00047192 ! larval feedforward local interneuron
+property_value: creation_date 2018-05-24T14:15:11Z xsd:dateTime
```

Comment:
Minor issue - looks like creation data annotation property still not set correctly in Protege.  See Go doc linked to previously.  Don't worry about manually changing anything added with this so far.  This can all be done with a single command in Protege.